.goreleaser.yml

```diff
@@ -30,23 +43,23 @@ dockers:
   dockerfile: Dockerfile
   goos: linux
   goarch: arm64
-  use: buildx
+  use: docker
```

Comment:
I think @bentito mentioned this was supposed to be reverted

Reply:
fully reverted now